tests/shouldwork/Cores/Xilinx/Floating/TH.hs

```diff
@@ -456,6 +492,16 @@ divBasicSamples =
   (_, maxExp) = floatRange (undefined :: Float)
   model (conditionFloat -> x) (conditionFloat -> y) = conditionFloat $ x / y
 
+compareBasicSamples :: [(Float, Float, F.Ordering)]
+compareBasicSamples =
+    delayOutput (natToNum @F.CompareDefDelay) F.NaN
+  $ [ (1.0, 2.0, F.LT)
+    , (2.0, 1.0, F.GT)
+    , (1.0, 1.0, F.EQ)
+    , (nan, 1.0, F.NaN)
```

Comment:
Other code in this module uses `F.xilinxNaN`, for consistency I think this should too.

Also, by importing `nan` it will give a name shadowing error in `nanTest`.